modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/whatsnew/WhatsNewViewModel.kt

```diff
@@ -13,29 +17,41 @@ import javax.inject.Inject
 import au.com.shiftyjelly.pocketcasts.localization.R as LR
 
 @HiltViewModel
-class WhatsNewViewModel @Inject constructor() : ViewModel() {
+class WhatsNewViewModel @Inject constructor(
+    settings: Settings,
+    feature: FeatureWrapper,
+) : ViewModel() {
     private val _state: MutableStateFlow<UiState> = MutableStateFlow(UiState.Loading)
     val state = _state.asStateFlow()
 
     private val _navigationState: MutableSharedFlow<NavigationState> = MutableSharedFlow()
     val navigationState = _navigationState.asSharedFlow()
 
     init {
+        val isBookmarksEnabled = FeatureFlag.isEnabled(feature.bookmarksFeature)
+
         _state.value = UiState.Loaded(
-            feature = WhatsNewFeature.AutoPlay,
+            feature = if (isBookmarksEnabled) WhatsNewFeature.Bookmarks else WhatsNewFeature.AutoPlay,
```

Comment:
This logic seems strange. With each new release if there is a what's new page wouldn't it be the new feature. I thought auto-play had already been released.

Reply:
What's new page is updated for a major release. It might span multiple releases until a user sees it. 
I just didn't discard What's new for AutoPlay since the next major release (bookmarks) is not enabled yet. Do you think I should discard it since it has been several releases since AutoPlay was released?

Reply:
That makes sense. Thanks for the explanation.